lib/pubnub.rb

```diff
@@ -4,6 +4,9 @@
 require 'open-uri'
 require 'openssl'
 require 'celluloid/current'
+
+Celluloid.boot
```

Comment:
This seems a bit surprising and maybe a bad idea—usually an app will require `celluloid/autostart`, which calls `Celluloid.boot` and registers the exit handler. Calling boot a second time will raise an a deadlock exception.